test/verify/check-services

```diff
@@ -194,6 +195,9 @@ WantedBy=default.target
 
         def svc_sel(service):
             return 'tr[data-goto-unit="%s"]' % service
+        def wait_systemctl_timer(time):
+            with testvm.Timeout(seconds=10, error_message="Timeout while waiting for systemctl list-timers"):
+                m.execute("cmd='systemctl list-timers'; until $cmd | grep -m 1 '%s'; do sleep 1; done" % time)
```

Comment:
m.execute("cmd='systemctl list-timers'; until $cmd | grep -m 1 '%s'; do _sleep 1_; done" % time) would loop and check if systemctl list-timers has the updated results. This keeps checking every second for a total of 10sec after which timeout occurs. 
@dperpeet which is better sleep 1 or sleep 0.5, here?